ras/lshw.py

```diff
@@ -207,8 +210,8 @@ def test_lshw_options(self):
             self.fail_cmd.append("lshw -enable network|grep -i interface")
 
         self.run_cmd("lshw -class -quiet")
-        if 'PowerVM'\
-                not in self.run_cmd_out("pseries_platform"):
+        if 'PowerNV'\
+                in self.run_cmd_out("pseries_platform"):
```

Comment:
this condition make this condition should not run in kvm so earlier condition seems me ok

Reply:
Got it - MY VMs were not started with -usb enabled meaning that HCI support wasn't there. So I will reverse the test, but I would also suggest to also check that usb is enabled otherwise test is irrelevant - what do you think ?

Reply:
@tfauck yes good idea